components/admin-dashboard/whitelisted-addresses.ts

```diff
@@ -1,4 +1,3 @@
-export const whitelistedAddresses = process.env
-  .NEXT_PUBLIC_WHITELISTED_ADDRESSES
-  ? process.env.NEXT_PUBLIC_WHITELISTED_ADDRESSES.split(",")
-  : [];
+import { env } from "../../config/env.mjs";
+
+export const whitelistedAddresses = env.NEXT_PUBLIC_WHITELISTED_ADDRESSES ?? [];
```

Comment:
why do you need a default value check here?

Reply:
I probably don't ':D